compiler/rustc_target/src/spec/targets/x86_64_unknown_nuttx.rs

```diff
@@ -0,0 +1,41 @@
+// Generic x86-64 target for bare-metal code - Floating point disabled
+//
+// Can be used in conjunction with the `target-feature` and
+// `target-cpu` compiler flags to opt-in more hardware-specific
+// features.
```

Comment:
Question: this comment seems... irrelevant. Possibly copy-pasta leftover?

Reply:
Sure, it's based on `x86_64-unknown-none`, I've modified the comment to reflect the actual change.

Thanks !